### ISSUE TITLE
Improper display output when "Increased Contrast" is enabled in OS X

### ISSUE THREAD

**Issue by SamRothCA** [SamRothCA/Today-Scripts#3]
When "Increased Contrast" is enabled in System Preferences > Accessibility > Display, various labels and text fields in Today Scripts will output black text rather than the default white, making them difficult to read. The widget will need to properly adopt the coloring patterns [as outlined in Apple's OS X Human Interface Guidelines](https://developer.apple.com/library/mac/documentation/UserExperience/Conceptual/OSXHIGuidelines/ColorTypography.html).

Credit to Reddit users [/u/amoeba](http://www.reddit.com/user/amoeba) and [/u/striata](http://www.reddit.com/user/striata) for finding this one.
